tests/integration/test_vulnerability_detector/test_windows/test_cpe_indexing.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import pytest
+import json
+from time import sleep
+from shutil import copy
+
+import wazuh_testing.vulnerability_detector as vd
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.file import truncate_file, compress_gzip_file
+from wazuh_testing.tools.services import control_service
+
+# Marks
+pytestmark = pytest.mark.tier(level=1)
+
+# Variables
+current_test_path = os.path.dirname(os.path.realpath(__file__))
+test_data_path = os.path.join(current_test_path, 'data')
+configurations_path = os.path.join(test_data_path, 'cpe_indexing.yaml')
+
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+
+# Set configuration
+parameters = [{'NVD_JSON_PATH': os.path.join(test_data_path, vd.CUSTOM_NVD_FEED)}]
+metadata = [{'nvd_json_path': os.path.join(test_data_path, vd.CUSTOM_NVD_FEED)}]
+ids = ['cpe_index_configuration']
+
+# Configuration data
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+
+system_data = [
+    {"target": "WINDOWS_SERVER_2013", "os_name": "Microsoft Windows Server 2003",
```

Comment:
It would be interesting to add an entry for `x64`

Reply:
Added I386 test in https://github.com/wazuh/wazuh-qa/pull/804/commits/08a63bc86942cc99777b8b739a4ea7887b0d2576

Reply:
It should be x86 instead of i386.